KayKnows-frontend/src/app/tree/tree.component.ts

```diff
@@ -16,4 +18,19 @@ export class TreeComponent implements OnInit {
         
     }
 
+    getColour(data: any): string {
+        return "3px solid #" + data.band_colour;
+    }
+
+    getBackgroundColour(data: any): string {
+        return "#" + data.band_colour;
```

Comment:
why did you store '#' as part of the hex number in the db as a string to avoid this?

Reply:
I fixed the database so it now includes hex